.github/workflows/cleanup-repo.yaml

```diff
@@ -1,7 +1,7 @@
 ﻿name: "Cleanup Repo"
 on:
   schedule:
```

Comment:
<!-- Consider adding a comment above the cron schedule to explain the reason for changing from '0 7 * * *' to '0 8 * * *'. This helps in understanding the decision for future reference. -->